grafana/.gitignore

```diff
@@ -1,3 +1,6 @@
 /grafana-6.4.3
 /grafana-6.4.3.tar.gz
 /grafana_webpack-6.4.3.f5f5731149f78e9d7039e56d7963f03bca74f948e40db979b59bbb2f1437dc47.tar.gz
+/grafana-6.4.3
```

Comment:
nit: these are duplicate

Reply:
they are. I could have sworn we fixed this so we didn't have to append new lines every time.

Reply:
The update.sh script just blindly appends them

Reply:
Note that in one of the other update.sh scripts I added the following
```
function gitignore() {
  local ignored=$1
  if [ -z "$(egrep '^$ignored$' .gitignore)" ] ; then
    echo "$ignored" >> .gitignore
  fi
}
```

Reply:
actually that code doesn't work properly but it was the right idea

Reply:
This is what it should be

```
function gitignore() {
  local ignored=$1
  if [ -z "$(egrep '^'$ignored'$' .gitignore)" ] ; then
    echo "$ignored" >> .gitignore
  fi
}
```

Reply:
and it looks as if this was added to the grafana update.sh but it didn't work because of the missing 's

Reply:
Should just be able to use a wildcard